.github/workflows/daily.yml

```diff
@@ -21,10 +21,13 @@ on:
   schedule:
     - cron: "0 0 * * *"
   workflow_dispatch:
+  pull_request: ## !!! TODO DELETED BEFORE MERGED
```

Comment:
TODO

After approved I will remove it. Now let's keep it as is to show the result.

Reply:
```suggestion
```